src/main/java/de/danielbechler/diff/selector/CollectionItemElementSelector.java

```diff
@@ -71,6 +94,6 @@ public boolean equals(final Object o)
 	@Override
 	public int hashCode()
 	{
-		return item != null ? item.hashCode() : 0;
+		return item != null ? identityStrategy.hashCode(item) : 0;
```

Comment:
Ok, thanks. :+1: 

> I think you are trying to protect the user from its own mistakes a bit too much.

By protecting the user I'm protecting myself from support requests and maintenance. :wink: